### PR TITLE
bigflow[base_frozen] extras to be used in pyproject.toml

### PR DESCRIPTION
**Pull request by beczkowb**

### CODE REVIEW

File: bigflow/_version.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
-__version__ = '1.4.2rc2'
+__version__ = '1.4.2rc3'
```
Comment (grzegorzwitkowski) [inline bigflow/_version.py:1]: shouldn't be just 1.4.2 if you are going to release a version 1.4.2?

Comment (beczkowb) [reply bigflow/_version.py]: I will release the final rc5, deploy it to the production in our system, then release 1.4.2

File: requirements/base_frozen.in [hunk 2]
```diff
@@ -0,0 +1,19 @@
+google-cloud-storage>=1.30,<2
+# Maintainers, please do not require google-api-core>=2.x.x
+# Until this issue is closed
+# https://github.com/googleapis/google-cloud-python/issues/10566
+MarkupSafe<2.1.0 # version 2.1.0 is broken
```
Comment (grzegorzwitkowski) [inline requirements/base_frozen.in:5]: there is a version 2.1.1 of MarkupSafe available. Does it fixe the issue?

**Review by iwko — APPROVED** [review]

File: requirements/base_frozen.in (hunk 2 above)
Comment (beczkowb) [reply requirements/base_frozen.in:5]: It's still broken. I have checked that.

**Pull request merged**